### DISCUSSION TITLE
Enhance reporting of errors from install script

### DISCUSSION THREAD

**Discussion by buddhikac96** [utPLSQL/utPLSQL#1339] [Feature Requests]
![image](https://user-images.githubusercontent.com/28675970/137272587-087153f6-ac47-4770-ae17-a125e85a81eb.png)

I tried to install the framework as SYS.

How can I resolve this?

**Comment by buddhikac96** [comment]
Finally, I found the reason (Actually my friend found the reason ;) ). 

Previously I was trying to execute the installation command from another location apart from where UTPLSQL scripts exist. But this works fine when I execute the command from the location where UTPLSQL exists.

**Comment by pesse** [comment]
@jgebal Maybe something we could fail-safe? Or give an expressive error message?
Don't know the capabilities of SQL*Plus enough - in SQLcl it would be possible.

**Comment by jgebal** [comment]
This was clearly an OS error.
One thing we could do is to make output even more verbose for the installation.
This might not be great idea by itself.
Alternatively we could try to do a check for current directory to see if it's OK.
That would be a nice little enhancement to add.
I'll reopen this issue so we can keep track of it and make an improvement.
